src/app/rosetta/README.md

```diff
@@ -362,7 +362,10 @@ The signer library used by the test agent can be used as a reference for further
 
 ### Rosetta CLI Validation
 
-The Data API is fully validated using the official `rosetta-cli` against private networks that issue every different type of transaction. There are no reconcilliation errors. We are in the middle of verifying reconcilliation errors against devnet.
+The Data API is fully validated using the official `rosetta-cli`
+against private networks that issue every different type of
+transaction. For instructions on how to run these tests manually,
+see `REAMDE-rosetta-cli.md`.
```

Comment:
Small typo here on the word `README`. Also, should this be linked to the respective file?

Reply:
Thanks, corrected.